src/coreclr/src/nativeaot/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -25,6 +25,10 @@ public static class RuntimeImports
     {
         private const string RuntimeLibrary = "[MRT]";
 
+        [DllImport(RuntimeLibrary, ExactSpelling = true, CallingConvention = CallingConvention.Cdecl)]
+        [SuppressGCTransition]
+        internal static extern IntPtr PalGetTickCount64();
```

Comment:
![](https://media.giphy.com/media/cQtlhD48EG0SY/giphy.gif)

Reply:
Yeah, I lost a patience to deal with this. It needs one-time initialization to work well. The one-time initialization code in corert was pain to port to C and it looks like it would not work on Apple Silicon anyway. So I have just wired this to `PalGetTickCount64` for now.

Reply:
Makes sense! Thanks!